lib/datadog/tracing/contrib/patcher.rb

```diff
@@ -55,6 +55,14 @@ def on_patch_error(e)
             Datadog.health_metrics.error_instrumentation_patch(1, tags: tags)
           end
 
+          def patch_results
+            @patch_results ||= nil
+          end
+
+          def patch_results=(patch_results)
+            @patch_results = patch_results
+          end
+
```

Comment:
This is equivalent to `attr_accessor :patch_results`: https://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/accessors.html#label:2 :
```suggestion
          attr_accessor :patch_results
```